### DISCUSSION TITLE
Default folder for saving subtitles

### DISCUSSION THREAD

**Discussion by ThomasSeeker** [smplayer-dev/smplayer#99] [General]
Hey there,

when I add subtitles via "Search Subtitles from OpenSubtitles.org"-Button ... I am asked to specify into which folder I want them to be saved. 

My wish would be, to not save them at all, but just display them. ... Is that possible?

Or if I want to save them, is it possible to change the default folder? (Right now it is /usr/bin.)

Best greetings!

**Comment by smplayer-dev** [comment]
The default folder is that same where the movie is, but if it's a online stream that may happen. Maybe I can change the folder to the temp folder or something.

**Comment by ThomasSeeker** [comment]
Ah, now I understand! ... Yes, this only occurs with online streams! ... Thank you for clarifying!

Yes, the solution for movies from the hard-drive is much better IMO than the one for streams. Because just a double-click is enough and the subtitles are downloaded.  ... but then they need to be deleted from the hard-drive afterwards.

The best solution for me would be: 

Have the choice to determine the default folder myself (for both stored and streamed movies). I would choose the temp folder most of the time, but sometimes another one (if I want to keep the subtitles).

The second best solution for me would be: 

If no choice is possible, I would like all subtitles (for both stored and streamed movies) go to /tmp/.

Is this possible?

And just to mention: I love SMplayer! ... :-)
